.github/workflows/scriptcheck.yml

```diff
@@ -17,12 +17,14 @@ jobs:
           sudo apt-get update
           sudo apt-get install shellcheck
           sudo apt-get install python3-setuptools
-          sudo pip3 install natsort
-          sudo pip3 install pexpect
-          sudo pip3 install pylint
-          sudo pip3 install unittest2
-          sudo pip install pytest
-          sudo pip3 install pytest
+          pip install --user pip --upgrade
```

Comment:
@firewave  Why you think it's better to install in user space here?

Reply:
Since the system wide packages are controlled by the package manager and installing (newer) outside packages can mess with the dependencies of the system ones and break some functionality. It is also suggested by `pip` itself.

Also upgrading `pip` to the latest version might also help with issues when installing newer packages. Might explain the `pylint` issues in the Windows CI builds which appear to be gone with this change.

These issue should be rare but I ran into them before - especially with distros which came with very outdated packages.

FYI I had this written up as a comment several days ago but somehow it got lost...

I have no idea why it no longer can find `pylint` but I had no time to look into it yet.